tasks/npmfix/index.js

```diff
@@ -26,20 +26,18 @@ function npmfix({packages} = {}) {
       const repoUrl = gitInfo.fromUrl(gitRemoteUrl).browse()
 
       return iter.parallel(lernaPackages, {log})((lernaPackage, log) => {
-        const moduleGitUrl =
-          repoUrl + '/tree/master/' + relative(process.cwd(), lernaPackage.location)
-
         return fs
           .readFile(lernaPackage, {log})('package.json', JSON.parse)
           .then(packageJson => {
             const updated = Object.assign({}, packageJson, {
-              homepage: moduleGitUrl,
+              homepage: repoUrl + '/tree/master/' + relative(process.cwd(), lernaPackage.location),
               dependencies: sortDependencies(packageJson.dependencies),
               devDependencies: sortDependencies(packageJson.devDependencies),
               peerDependencies: sortDependencies(packageJson.peerDependencies),
               repository: {
                 type: 'git',
-                url: moduleGitUrl
+                url: gitRemoteUrl,
+                directory: '/' + relative(process.cwd(), lernaPackage.location)
```

Comment:
for root-located packages maybe we should skip `directory` property at all?
WDYT?

Reply:
@hugebdu do we change smth in the root `package.json`?
Just realize, that there is no ability to have several packages at the root folder (several package.json in one folder?)
So every package will be inside `/itsNameFolder`
Even when there will be no `packages` folder

Reply:
indeed. good point